Product-Category/Family-category.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Trendy-T</title>
+    <link rel="stylesheet" href="/Product-Category/product-category.css">
+    <link rel="stylesheet" href="/Product-Category/product-category-responsive.css">
+</head>
+
+<body>
+    <!-- preloader -->
+
+    <div class="preloader"></div>
+
+    <!-- Navigation Bar Desktop -->
+    <nav class="nav-bar nav-bar--simple">
+        <nav class="nav-bar--logo">
+            <a href="/index.html"><img class="nav-bar__img--logo" src="/assets/image/HomePage-Images/Trendy-T Logo.png"
+                    alt="AQUA_UI"></a>
+        </nav>
+        <ul class="nav-bar__list">
+            <li><a class="nav-bar__item" href="/index.html">Home</a></li>
+            <li><a class="nav-bar__item" href="/Product-Page/product.html">Product</a></li>
+
+            <li class="nav-bar__item"><a href="/wishlist-page/wishlist.html"><img
+                        class="nav-bar__img--icon nav-bar__img--wishlist"
+                        src="/assets/image/HomePage-Images/Wishlist.png" alt="Wishlist"></a></li>
+
+            <li class="nav-bar__item badge__wrapper"><a href="/Cart-Page/cart.html"><img
+                        src="/assets/image/HomePage-Images/cart.png" alt="cart-badge"
+                        class="nav-bar__img--icon nav-bar__img--cart badge__icon--img"></a>
+                <span class="badge__icon--primary badge--logo ">1</span>
+            </li>
+
+            <li class="nav-bar__item"><a href="/Login-page/login.html"><img
+                        class="nav-bar__img--icon nav-bar__img--login" src="/assets/image/HomePage-Images/Login.png"
+                        alt="Login/Sign-Up"></a></li>
+        </ul>
+    </nav>
+
+    <!-- Navigation Bar For Tab and Mobile  -->
+
+    <div class="nav-bar nav-bar--hamburger">
+        <div class="nav-bar--logo">
+            <a href="/index.html"><img class="nav-bar__img--logo" src="/assets/image/HomePage-Images/Trendy-T Logo.png"
+                    alt="TRENDY-T"></a>
+        </div>
+        <div class="nav-bar--hamburger">
+            <img class="nav-bar__img--hamburger" src="/assets/image/HomePage-Images/Hamburger Menu.png" alt="Hamburger">
```

Comment:
Do use / before closing img tag

Reply:
Do I need to use / before closing img tag in javaScript?